client/gutenberg/editor/main.jsx

```diff
@@ -16,6 +16,8 @@ import { getSelectedSiteId } from 'state/ui/selectors';
 import { getSiteSlug } from 'state/sites/selectors';
 import { overrideAPIPaths } from './utils';
 
+import 'gutenberg/extensions/presets/o2/editor';
```

Comment:
@dmsnell could you leave a TODO to remind us which blocks we should include for which environments or sites? The `trials` block for example wouldn't make a ton of sense for all of wpcom to be able to use.

Reply:
I think it would make sense to create a separate preset for the extensions we'll want to use on .com and open to the public. Production would use that preset, and the rest of the envs would use the other preset with the additional blocks. What do you think @dmsnell?

Reply:
yes, or ideally run the `jetpack` preset in Calypso :wink:

I have added the *DO NOT MERGE* flag here to help make sure we don't deploy this code as is